Install data files to FHS data dir, and doc files to FHS doc dir

diff --git i/Makefile.am w/Makefile.am
index 31529a0..957870c 100644
--- i/Makefile.am
+++ w/Makefile.am
@@ -5,8 +5,8 @@ SUBDIRS = src
 
 EXTRA_DIST = apvlv.1 apvlvrc.example Startup.pdf icons/*
 
-pdf_DATA = Startup.pdf
-doc_DATA = apvlvrc.example icons/reg.png icons/dir.png icons/pdf.png
+doc_DATA = Startup.pdf apvlvrc.example
+pkgdata_DATA = icons/reg.png icons/dir.png icons/pdf.png
 man_MANS = apvlv.1
 
 install-exec-local:
diff --git i/configure.ac w/configure.ac
index d9d951e..70d3d98 100644
--- i/configure.ac
+++ w/configure.ac
@@ -73,6 +73,9 @@ AC_ARG_WITH(docdir,
 AC_SUBST(DOCDIR)
 AC_DEFINE_UNQUOTED(DOCDIR, ["$docdir"], [The documents directory])
 
+AC_SUBST(DATADIR)
+AC_DEFINE_UNQUOTED(DATADIR, ["$datadir/$PACKAGE_TARNAME"], [The data directory])
+
 mandir="$prefix/man"
 AC_ARG_WITH(mandir,
             [[  --with-mandir=DIR       directory for man page of apvlv [PREFIX/man]]], 
diff --git i/src/ApvlvUtil.cpp w/src/ApvlvUtil.cpp
index eb8aa37..809d762 100644
--- i/src/ApvlvUtil.cpp
+++ w/src/ApvlvUtil.cpp
@@ -46,7 +46,8 @@ namespace apvlv
 {
 
 #ifdef WIN32
-  string helppdf = "~\\Startup.pdf";
+  string eAC_DEFINE_UNQUOTED(DOCDIR, ["$docdir"], [The documents directory])
+elppdf = "~\\Startup.pdf";
   string iniexam = "~\\apvlvrc.example";
   string iconreg = "~\\reg.png";
   string icondir = "~\\dir.png";
@@ -56,9 +57,9 @@ namespace apvlv
 #else
   string helppdf = string (DOCDIR) + "/Startup.pdf";
   string iniexam = string (DOCDIR) + "/apvlvrc.example";
-  string iconreg = string (DOCDIR) + "/reg.png";
-  string icondir = string (DOCDIR) + "/dir.png";
-  string iconpdf = string (DOCDIR) + "/pdf.png";
+  string iconreg = string (DATADIR) + "/reg.png";
+  string icondir = string (DATADIR) + "/dir.png";
+  string iconpdf = string (DATADIR) + "/pdf.png";
   string inifile = "~/.apvlvrc";
   string sessionfile = "~/.apvlvinfo";
 #endif
